DocumentFormat.OpenXml/src/ofapi/Validation/ValidationResources.resx

```diff
@@ -113,21 +112,11 @@
     <value>2.0</value>
   </resheader>
   <resheader name="reader">
-    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+    <value>System.Resources.ResXResourceReader, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
   <resheader name="writer">
-    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
+    <value>System.Resources.ResXResourceWriter, System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
   </resheader>
-  <assembly alias="System.Windows.Forms" name="System.Windows.Forms, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" />
-  <data name="O12SchemaConstraintDatas" type="System.Resources.ResXFileRef, System.Windows.Forms">
-    <value>..\..\generatedcode\o12schemaconstraintdatas.bin;System.Byte[], mscorlib, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
-  </data>
-  <data name="O14SchemaConstraintDatas" type="System.Resources.ResXFileRef, System.Windows.Forms">
-    <value>..\..\generatedcode\o14schemaconstraintdatas.bin;System.Byte[], mscorlib, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
-  </data>
-  <data name="O15SchemaConstraintDatas" type="System.Resources.ResXFileRef, System.Windows.Forms">
-    <value>..\..\generatedcode\o15schemaconstraintdatas.bin;System.Byte[], mscorlib, Version=2.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089</value>
```

Comment:
These file names lead to build errors on Travis CI because their case differs from the actual case of the corresponding files.